<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Дейкстра</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>641</width>
     <height>481</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string notr="true">Головна</string>
    </attribute>
    <widget class="QLabel" name="dijkstra_l_number">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Кількість точок:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="dijkstra_i_number">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>140</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="dijkstra_generated">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>70</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Генерувати</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="dijkstra_connectors_table">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>120</y>
       <width>256</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="dijkstra_go">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>332</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Виконати</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="dijkstra_i_m_number">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>40</y>
       <width>140</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="dijkstra_l_m_number">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>20</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Кількість зєднань:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="dijkstra_i_start">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>40</y>
       <width>140</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="dijkstra_l_start">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>20</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Початкова точка:</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Результат роботи</string>
    </attribute>
    <widget class="QPlainTextEdit" name="dijkstra_log">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>621</width>
       <height>451</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
